core/src/progress_map.rs

```diff
@@ -456,8 +464,12 @@ impl ProgressMap {
             .and_then(|fork_progress| fork_progress.fork_stats.bank_hash)
     }
 
-    pub fn is_propagated(&self, slot: Slot) -> bool {
-        let leader_slot_to_check = self.get_latest_leader_slot(slot);
+    pub fn is_propagated(&self, slot: Slot, slot_must_exist: bool) -> bool {
```

Comment:
Looks like `slot_must_exist` is always `true`. Want to just make this `is_propagated_assert_slot_must_exist` and remove the flag?

Reply:
done